.github/workflows/scheduled-run-all-tests-workflow.yaml

```diff
@@ -53,7 +54,7 @@ jobs:
         with:
           firefox-version: "latest-esr"
       - name: "Check docker status"
-        run: systemctl status docker
+        run: "systemctl status docker"
```

Comment:
@spriadka is it correct?

Reply:
nope, we are now placing tests into the submodules listed. Before we used to run `--specs=cypress/e2e/*${MODULE_NAME}*` now we are running `--specs=cypress/e2e/${MODULE_NAME}/*cy.ts`. So just get rid of the asterisks and it will be okay 🙂